src/components/article-body/body-components/paragraph.tsx

```diff
@@ -0,0 +1,13 @@
+import { ComponentFactory } from '../component-factory';
+import React from 'react';
+import { ArticleNode } from '../../../interfaces/article-body-node';
+
+export const Paragraph = ({ children, ...rest }: { children: any }) => {
+    return (
+        <>
+            {children.map((child: ArticleNode, index: number) => (
+                <ComponentFactory {...rest} key={index} nodeData={child} />
```

Comment:
Don't use the index as the key as it could cause issues in React https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067